extensions/skins/conversation_v1/Conversation.js

```diff
@@ -730,6 +651,7 @@ oppia.directive('progressDots', [function() {
       });
 
       $scope.changeActiveDot = function(index) {
+        $scope.progressDotClicked = true;
```

Comment:
All this seems a bit clumsy. Is there a way to keep track of this entirely within the directive, so that all the animation can be fully handled local to this directive? It seems to me like it should be conceptually possible -- if the current dot index is the same as the number of dots, and a new dot shows up, animate it. I think this is already detected in the $scope.$watch.

Or maybe I'm missing something here?